share/cmake/modules/FindExpat.cmake

```diff
@@ -59,12 +59,20 @@ if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
     # Attempt to find static library first if this is set
     if(EXPAT_STATIC_LIBRARY)
         set(_EXPAT_STATIC "${CMAKE_STATIC_LIBRARY_PREFIX}expat${CMAKE_STATIC_LIBRARY_SUFFIX}")
+        if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL Debug)
```

Comment:
In the CMake systems we depend on, the different suffix only applies when doing a full Debug build. This is why I used Debug as the condition check instead of Release, so that the Release behavior would apply to the other non-Debug types. I will double check that though with each project.

Reply:
1. ok.
2. What about `CMAKE_BUILD_TYPE` equal to `debug`(i.e. all in lower case)? 
The `STREQUAL` is case-sensitive.

Reply:
Fixed